wapitiCore/attack/mod_csp.py

```diff
@@ -65,21 +117,14 @@ async def attack(self, request: Request, response: Optional[Response] = None):
             )
         else:
             csp_dict = csp_header_to_dict(response.headers["Content-Security-Policy"])
-
+            info = ""
             for policy_name in CSP_CHECK_LISTS:
```

Comment:
It is convenient to have all policy names to check in `CSP_CHECK_LISTS` at this line, however it is very misleading that some keys correspond to safe values and the other to unsafe values.

It would be nice to split `CSP_CHECK_LISTS` in two dictionaries to differentiate the cases and prevent an error to come.

CSP is not easy to get so the more explicit the code, the lower the risk of introducing bugs